rules/rules.libsonnet

```diff
@@ -0,0 +1,17 @@
+{
+  prometheusRules+:: {
+    groups+: [
+      {
+        name: 'ceph.rules',
+        rules: [
+          {
+            record: 'ceph_node_down',
```

Comment:
we need to name this according to the convention.

Reply:
rules.libsonnet should ideally have only the imports. This rule should be in some other file, say 'node.libsonnet'

Reply:
ack, will change